scripts/ci/in_container/entrypoint_ci.sh

```diff
@@ -211,6 +211,10 @@ fi
 
 TEST_DIR="tests/"
 
+if [[ ${#@} -gt 0 && -n "$1" ]]; then
+    TEST_DIR="$1"
+fi
+
 if [[ -n ${RUNTIME} ]]; then
     CI_ARGS+=("--runtime" "${RUNTIME}" "-rpfExX")
     TEST_DIR="tests/runtime"
```

Comment:
```suggestion
    TESTS_TO_RUN="tests/runtime"
```

Reply:
And also we should change TEST_DIR below to TESTS_TO_RUN :)

Reply:
fixed!!! https://github.com/apache/airflow/pull/7396/commits/f6be34fd8725ae44b156f0e51076d92bf69a36a0